app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -80,6 +91,20 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
         mode: mode,
       };
 
+      if (!command.capacity) {
+        command.capacity = {
+          min: command.targetSize,
+          max: command.targetSize,
+          desired: command.targetSize,
+        };
+      }
+
+      if (!command.scalingPolicy) {
```

Comment:
This whole block can be shortened to `_.set(command, 'scalingPolicy.cpuUtilization.target', 40)`